packages/core/package.json

```diff
@@ -37,6 +37,7 @@
     "prepack": "yarn build"
   },
   "devDependencies": {
+    "@cardano-sdk/util-dev": "workspace:~",
```

Comment:
This creates a circular dependency. I suggest following the existing pattern to use relative import instead (see `packages/core/test/Cardano/Address/Address.test.ts`). There must be a better way to do it, but for now that's what we have.

Reply:
Ah, I didn't check if util-dev had a dependency on core, my bad (I was expecting it not to)

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1076/commits/d747a9da096e4db7f5c85e34d9afbb7b01f697b4